autoria.py

```diff
@@ -0,0 +1,242 @@
+"""Python auto.ria.com API.
+
+Python implementation of API intended for calulating
+average used car prices that are sold on http://auto.ria.com
+"""
+
+
+import requests
+import json
+from fnmatch import fnmatch
+
+
+def make_request(url: str) -> json:
```

Comment:
Result is also `str`, not "json" - json is a module, it can not be return type here.